tools/luci/lib/luci_common.dart

```diff
@@ -0,0 +1,400 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Probably because this file is some combination of `environment.dart`, `utils.dart`, and `exceptions.dart`.

Reply:
In the last 6 months I put a lot of effort to split these files and keep every file small. Please split them otherwise I cannot approve this change. (Since I'm the person who works most using these files, and would probably would lose the most productivity over these huge files, I think my request being blocking makes sense :) )

Reply:
Done.